server/controllers/Bookhandlers.js

```diff
@@ -32,7 +57,7 @@ exports.allBooks = (req, res) => {
     if (allBooks === null || allBooks.length === 0) { // if no book is found
       res.status(200).json({
         status: 'unsuccessful',
-        message: 'No Books Fam',
+        message: 'No Books',
```

Comment:
if no book found should be a 404 status code. Please take some time to research on appropriate status codes.